bitnami/grafana-loki/Chart.yaml

```
@@ -44,4 +44,4 @@ name: grafana-loki
 sources:
   - https://github.com/bitnami/bitnami-docker-grafana-loki
   - https://github.com/grafana/loki/
-version: 2.1.2
+version: 2.2.2
```

I think there is a typo here:
```suggestion
version: 2.2.0
```